packages/odyssey/src/scss/components/_modal.scss

```diff
@@ -15,7 +15,7 @@
 .ods-modal--overlay {
   display: flex;
   position: fixed;
-  z-index: 1;
+  z-index: 10;
```

Comment:
This needs to be checked once in client products to make sure this is not breaking.